README.md

```diff
@@ -26,4 +24,11 @@ The plan for Hazel is two-fold: to create a powerful 3D engine, but also to serv
 - Integrated 3rd party 2D and 3D physics engine
 - Procedural terrain and world generation
 - Artificial Intelligence
-- Audio system
\ No newline at end of file
+- Audio system
+
```

Comment:
Maybe add a section "Short term goals" and add in that we want to be able to use Hazel to create a basic game by the end of 2020, bit like how he explained it in the video... Don't forget to add a small disclamer that this is subject to change at any time